content/en/vendors/_index.md

```diff
@@ -1,6 +1,5 @@
 ---
-title: "Vendor Support"
-weight: 50
```

Comment:
This weight is useless since the page doesn't appear in any (top- or side-) nav.